surmise/emulationmethods/PCGPwImpute.py

```diff
@@ -70,10 +70,13 @@ def fit(fitinfo, x, theta, f, epsilonPC=0.001, lognugmean=-10,
     if not np.all(np.isfinite(f)):
         fitinfo['mof'] = np.logical_not(np.isfinite(f))
         __completef(fitinfo, compmethod=completionmethod)
-        print('Completing f with method IterativeImputer:{:s}.'.format(completionmethod))
+        if verbose > 0:
+            print('Completing f with method IterativeImputer:{:s}.'.format(completionmethod))
```

Comment:
At some point (not now!) you might consider adding a logger functionality to your design.